examples/src/main/java/org/apache/spark/examples/sql/JavaSQLDataSourceExample.java

```diff
@@ -21,6 +21,7 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import java.util.Properties;
 // $example off:schema_merging$
 
```

Comment:
Oh, maybe, my previous comment was not clear. I meant

```java
Import java.util.List;
// $example off:schema_merging$
Import java.util.Properties;
```

I haven't tried to build the doc against the current state but I guess we won't need this import for Parquet`s schema mering example.